docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -57,17 +57,21 @@ when migrating between versions.
 
 A snapshot contains a copy of the on-disk data structures that make up an
 index or a data stream's backing indices. This means that snapshots can only be restored to versions of
-{es} that can read the indices:
-
-* A snapshot of an index created in 6.x can be restored to 7.x.
-* A snapshot of an index created in 5.x can be restored to 6.x.
-* A snapshot of an index created in 2.x can be restored to 5.x.
-* A snapshot of an index created in 1.x can be restored to 2.x.
-
-Conversely, snapshots of indices created in 1.x **cannot** be restored to 5.x
-or 6.x, snapshots of indices created in 2.x **cannot** be restored to 6.x
-or 7.x, and snapshots of indices created in 5.x **cannot** be restored to 7.x
-or 8.x.
+{es} that can read the indices.
+
+The following table indicates snapshot compatibility between versions.
+
+// tag::snapshot-compatibility-matrix[]
+[cols="6*^"]
+|===
+| Compatibility | 2.x | 5.x | 6.x | 7.x | 8.x
+| 1.x |{yes-icon} |{no-icon}  |{no-icon}  |{no-icon}  |{no-icon}
+| 2.x |{no-icon}  |{yes-icon} |{no-icon}  |{no-icon}  |{no-icon}
+| 5.x |{no-icon}  |{no-icon}  |{yes-icon} |{no-icon}  |{no-icon}
+| 6.x |{no-icon}  |{no-icon}  |{no-icon}  |{yes-icon} |{no-icon}
+| 7.x |{no-icon}  |{no-icon}  |{no-icon}  |{no-icon}  |{yes-icon}
+|===
+// end::snapshot-compatibility-matrix[]
 
 We do not recommend restoring snapshots from later {es} versions in earlier
```

Comment:
Sorry for mixing this in here but it relates. This sentence is not correct. ES does not support restoring a newer version snapshot to an older version cluster at all. It's just not possible. So "We do not recommend " -> "Elasticsearch does not support " or so :)

Reply:
Changing the wording to be stronger:
>IMPORTANT: Restoring snapshots from later Elasticsearch versions to earlier versions of Elasticsearch is not supported. For example, a snapshot taken in 7.6.0 cannot be restored to 7.5.0.